src/main/java/com/epam/esm/repository/TagRepository.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.esm.repository;
+
+import com.epam.esm.entities.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+
+import java.util.List;
+
+@org.springframework.stereotype.Repository
+public class TagRepository implements Repository<Tag> {
+
+    public static final String SELECT_BY_ID = "SELECT * FROM tag WHERE id=?";
+    public static final String GET_ALL = "SELECT * FROM tag";
+    private JdbcTemplate jdbcTemplate;
+    private RowMapper<Tag> mapper;
```

Comment:
Are you going to change the values for these reference variables?

Reply:
Made them final and used Constructor injection